### DISCUSSION TITLE
Components to vector variable

### DISCUSSION THREAD

**Discussion by am-tc01** [idaholab/moose#20745] [Q&A General] [answered]
Hi,
I am transfering a vector variable from a main app to a sub app. For that I have to split the variable into its components using `VectorVariableComponentAux`. My question: is there a way to somehow combine the components in the sub app to make a single vector variable?
Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

I couldnt find one in the docs. 
But it shouldnt be too hard to make really.
Look at the VectorFunctionAux for an example of a vectorAuxKernel,
and the SelfAux for an example of a kernel that just copies a variable. 

Guillaume
